docs/source/en/model_doc/whisper.md

```diff
@@ -35,6 +35,85 @@ Tips:
 This model was contributed by [Arthur Zucker](https://huggingface.co/ArthurZ). The Tensorflow version of this model was contributed by [amyeroberts](https://huggingface.co/amyeroberts).
 The original code can be found [here](https://github.com/openai/whisper).
 
+## Inference
+
+Here is a step-by-step guide to transcribing an audio sample using a pre-trained Whisper model:
+
+```python
+>>> import torchaudio
```

Comment:
We should avoid using extra dependencies like `torchaudio` in our code snippets - could you maybe refactor this to only use dependencies in the Transformers library?

Reply:
Changed to use `datasets.load_dataset`.